src/app/pages/OnDemandRadioPage/index.jsx

```diff
@@ -4,12 +4,21 @@ import styled from 'styled-components';
 import path from 'ramda/src/path';
 import MetadataContainer from '../../containers/Metadata';
 import Grid, { GelPageGrid } from '#app/components/Grid';
-import OnDemandRadio from '#containers/RadioPageBlocks/Blocks/OnDemandRadio';
 import { ServiceContext } from '../../contexts/ServiceContext';
 
+import HeadingBlock from '#containers/RadioPageBlocks/Blocks/Heading';
+import ParagraphBlock from '#containers/RadioPageBlocks/Blocks/Paragraph';
+
+const SKIP_LINK_ANCHOR_ID = 'content';
+
 const OnDemandRadioPage = ({ pageData }) => {
+  const blocks = path(['content', 'blocks'], pageData);
```

Comment:
What if we were to do something like this:

```
const episode = path(['content', 'blocks', 0], pageData);
const { title: episodeTitle } = episode;
const summary = path(['synopses', 'short'], episode);
```

Reply:
Please can you explain why

Reply:
My reasoning was - I had a look at the JSON and there only appears to be 1 block within content.blocks - which is the episode & all relevant info we need for now.  I thought it would make the code a little bit more readable, but would like a second/third opinion if possible

Reply:
My opinion would be to do what Jonathan suggested here - it would negate this, and simplify things more generally: https://github.com/bbc/simorgh/pull/5934#pullrequestreview-378494689

ie, define our own schema for `pageData` that only has the fields we want, in the format we want.  We then wouldn't have to deal with the weird `blocks` format at all in any components